CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ set_property(CACHE RMM_LOGGING_LEVEL PROPERTY STRINGS "TRACE" "DEBUG" "INFO" "WA
                                               "CRITICAL" "OFF")
 
 # Set logging level. Must go before including gtests and benchmarks. Set the possible values of
-# build type for cmake-gui
+# build type for cmake-gui.
```

Comment:
Making trivial changes to these files anywhere other than the copyright statements, to convince the hook that they really were modified.

Without those changes, it passes locally but fails in CI like this:

```text
In file CMakeLists.txt:2:17:
 # Copyright (c) 2018-2024, NVIDIA CORPORATION.
warning: copyright is not out of date and should not be updated
```

([build link](https://github.com/rapidsai/rmm/actions/runs/9116138589/job/25064066207))

That's clearly wrong: 

<img width="1047" alt="Screenshot 2024-05-16 at 12 05 11 PM" src="https://github.com/rapidsai/rmm/assets/7608904/ed6e5550-c23f-4d67-967d-2900ad9ffe9a">

https://github.com/rapidsai/rmm/commits/branch-24.06/CMakeLists.txt

Reply:
> Where do you see the files being updated?

@KyleFromNVIDIA I was typing this at the same time you were typing https://github.com/rapidsai/rmm/pull/1564#issuecomment-2115781662.

Reply:
If you agree with me that this seems like a bug in the hook and not some mistake I'm making, I can put up a report at https://github.com/rapidsai/pre-commit-hooks/issues.

Reply:
We talked about this offline. I'll put up a report in the repo where this pre-commit hook is maintained and link it here.

Long story short... my fork of `rmm` was about a month behind this repo, and that led to my local `pre-commit` and CI disagreeing.

Reply:
> *I'll put up a report ... and link it here*

https://github.com/rapidsai/pre-commit-hooks/issues/32